spec/factories/courses.rb

```diff
@@ -1,6 +1,7 @@
 FactoryBot.define do
   factory :course do
-    name { "#{Faker::Job.unique.field} 101" }
+    :institute
```

Comment:
Drop the leading colon

```suggestion
    institute
```

Reply:
🤦